server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -364,133 +445,128 @@ public ClusterMetadataManifest writeIncrementalMetadata(
         if (durationMillis >= slowWriteLoggingThreshold.getMillis()) {
```

Comment:
can we generate the string for logging in common way, otherwise there is duplication

Reply:
Ack